src/main/java/fact/utils/ElementwiseMultiplyDoubleArray.java

```diff
@@ -0,0 +1,46 @@
+package fact.utils;
+
+import stream.Data;
+import stream.Processor;
+import stream.annotations.Parameter;
+import fact.features.singlePulse.timeSeriesExtraction.ElementWise;
```

Comment:
Wrong place for this comment? 
I guess you are taking up on a discussion you had with the author of Elementwise?